### PR TITLE
fix: Trigger updates for mysqlinstance + sqlserverinstance mockgcp tests

### PR DESCRIPTION
**Pull request by jasonvigil**
### Change description

Seems that the existing `metadata.labels` changes do not incur updates to the resources. So that we exercise the mockgcp update path, made some changes to `spec.settings.diskSize` for these resources.

### CODE REVIEW

**Comment by jasonvigil** [comment]
WIP: the disk size field isn't updating

File: pkg/test/resourcefixture/testdata/basic/sql/v1beta1/sqlinstance/mysqlinstance/_http.log [hunk 1]
```diff
@@ -176,8 +177,8 @@ Grpc-Metadata-Content-Type: application/grpc
     "cert": "-----BEGIN CERTIFICATE-----\n-----END CERTIFICATE-----\n",
     "certSerialNumber": "0",
     "commonName": "common-name",
-    "createTime": "2024-06-19T22:54:22.309289161Z",
-    "expirationTime": "2025-06-19T22:54:22.309289161Z",
+    "createTime": "2024-06-20T23:54:19.971010960Z",
+    "expirationTime": "2025-06-20T23:54:19.971010960Z",
```
Comment (jasonvigil) [inline pkg/test/resourcefixture/testdata/basic/sql/v1beta1/sqlinstance/mysqlinstance/_http.log:181]: @yuwenma should we normalize these timestamps?

Comment (jasonvigil) [reply pkg/test/resourcefixture/testdata/basic/sql/v1beta1/sqlinstance/mysqlinstance/_http.log]: Looks like this was fixed in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2094

**Review by yuwenma — COMMENTED** [review]
/lgtm
/approve

Looks good! Is those two fields the only mutable fields? 
FYI, we don't need to verify each label changes (that logic is independent from the Direct controller code). But no harm.

**Pull request merged**